raiden/ui/cli.py

```diff
@@ -306,4 +306,4 @@ def run(ctx, **kwargs):
         gevent.signal(signal.SIGINT, event.set)
         event.wait()
 
-        app_.stop()
+        app_.stop(graceful=True)
```

Comment:
> We should only have it when we want to explicitly test for that scenario. So the default fixture value should be `false. I know it's a bit more work than manually setting it to false in the app constructor in the one test but perhaps in the future there will be more tests or there are gonna be some other fixtures

Let me clarify this: the moment the `cleanup` function is called, we already gave up control of our test. So I can not think of **any** scenario where the cleanup should need to do the `graceful` shutdown as it stands.